pyproject.toml

```diff
@@ -1,59 +1,68 @@
-[tool.poetry]
+[project]
 name = "poetry"
 version = "2.0.0"
 description = "Python dependency management and packaging made easy."
-authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+requires-python = ">=3.9,<4.0"
+dependencies = [
+    "poetry-core (==2.0.0)",
```

Comment:
1. why are we pinning to a version? IIRC before 2.0 we had a `^` pin on the core.
2. are the parenthesis fine with the spec? (First I see those used)

Reply:
1. We do this since Poetry 1.3. IIRC, the idea behind this was that we will do a Poetry release anyway even if there is just a poetry-core bugfix release.
2. I think so. That is what Poetry procudes for core metadata.